gui/src-tauri/Cargo.toml

```diff
@@ -23,19 +23,23 @@ custom-protocol = ["tauri/custom-protocol"]
 [build-dependencies]
 tauri-build = { version = "1.2", features = [] }
 cfg_aliases = "0.1"
+shadow-rs = "0.18"
```

Comment:
thats bindings for libgit2 tho, idk how to show current tag with ``git describe``

Reply:
it does not error out if there is no tag, I'm doing this already in the overlay. If there is no tag, it will use the commit hash

Reply:
Ok I understand how ``git describe`` works, the default config of git_version is nice because it appends ``-modified`` if it's dirty. So ``git describe`` literally what it does is describe the current commit, it will use a tag apparently when it's used on a clean commit with a tag or if not it will use the hash.
My currenty implementation is by getting both separately and all versions on all parts look like this:
```
slimevr.exe -V   
slimevr git-8e5e142c
```
if a tag exists:
```
slimevr.exe -V   
slimevr TAG_NAME
```

The ``git_version!`` one will look like
```
slimevr.exe -V   
slimevr 8e5e142c-modified
```
if a tag exists (and it's not dirty):
```
slimevr.exe -V   
slimevr TAG_NAME
```
My version of the repo will always be dirty because of the submodules :sob:

Reply:
see review comment

Reply:
resolving, as we decided to use shadow.